src/geom/intersects/LineToLine.js

```diff
@@ -17,13 +15,12 @@ var Point = require('../point/Point');
  *
  * @param {Phaser.Geom.Line} line1 - The first Line to check.
  * @param {Phaser.Geom.Line} line2 - The second Line to check.
- * @param {Phaser.Geom.Point} [out] - A Point in which to optionally store the point of intersection.
+ * @param {(object|Phaser.Geom.Point|Phaser.Math.Vector2)} [out] - An optional Point-like object in which to store the coordinates of intersection, if needed.
```

Comment:
I think `Phaser.Types.Math.Vector2Like` is best here.

Reply:
Done, I think.
I pushed another commit to my forked repository, but I'm not sure if this is the right way: this is my second PR ever! 🤦‍♂️